### ISSUE TITLE
component throws deprecation warning in react 15.5

### ISSUE THREAD

**Issue by ZebraFlesh** [omniscientjs/omniscient#134]
Omniscient components created under react 15.5 display the following deprecation warning:

```Warning: MyComponent: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.```

Using `createReactClass` instead of `React.createClass` should resolve this issue.

**Comment by neftaly** [comment]
@mikaelbr v4.1.1 on NPM doesn't seem to have this fix applied, did you build before publishing?

Suggest you add this to `scripts` in package.json:
```json
"prepublish": "npm test && npm run build && npm run build-docs"
```

**Comment by dpoindexter** [comment]
@mikaelbr, it looks like v4.2.0 is still marked as WIP in the changelog. Are there plans to add other fixes, or can v4.2.0 be released to support React v16.x?

**Comment by mikaelbr** [comment]
I had some big plans in internal rewrites, but haven't gotten around to it. Will happily and joyfully accept PR with 16.x support! 🎉
